wagtail/contrib/redirects/templates/wagtailredirects/results.html

```diff
@@ -1,23 +1,19 @@
+{% extends "wagtailadmin/generic/index_results.html" %}
 {% load i18n %}
-{% if redirects %}
-    {% if query_string %}
-        <h2 role="alert">
-            {% blocktrans trimmed count counter=redirects.paginator.count %}
-                There is {{ counter }} match
-            {% plural %}
-                There are {{ counter }} matches
-            {% endblocktrans %}
-        </h2>
-    {% endif %}
 
-    {% include "wagtailredirects/list.html" %}
-
-    {% include "wagtailadmin/shared/pagination_nav.html" with items=redirects linkurl="wagtailredirects:index" %}
-{% else %}
+{% block no_results_message %}
     {% if query_string %}
-        <p role="alert">{% blocktrans trimmed %}Sorry, no redirects match "<em>{{ query_string }}</em>"{% endblocktrans %}</p>
+        <p role="alert">
+            {% blocktrans trimmed %}
+                Sorry, no redirects match "<em>{{ query_string }}</em>"
+            {% endblocktrans %}
+        </p>
     {% else %}
         {% url 'wagtailredirects:add' as wagtailredirects_add_redirect_url %}
-        <p>{% blocktrans trimmed %}No redirects have been created. Why not <a href="{{ wagtailredirects_add_redirect_url }}">add one</a>?{% endblocktrans %}</p>
+        <p>
+            {% blocktrans trimmed %}
+                No redirects have been created. Why not <a href="{{ wagtailredirects_add_redirect_url }}">add one</a>?
+            {% endblocktrans %}
+        </p>
     {% endif %}
-{% endif %}
+{% endblock no_results_message %}
```

Comment:
We generally don't use the block name in `endblock` tags. Also, please make sure that the files end with a new line.

```suggestion
{% endblock %}

```